Bigtable/src/Instance.php

```diff
@@ -0,0 +1,271 @@
+     */
+    public function reload(array $options = [])
+    {
+        throw new \BadMethodCallException('This method is not implemented yet');
+    }
+
+    /**
+     * Return the instance state.
+     *
+     * When instances are created or updated, they may take some time before
+     * they are ready for use. This method allows for checking whether an
+     * instance is ready.
+     *
+     *
+     * @param array $options [optional] Configuration options.
+     */
+    public function state(array $options = [])
+    {
+        throw new \BadMethodCallException('This method is not implemented yet');
+    }
+
+    /**
+     * Create a new instance.
+     *
+     * Example:
+     * ```
+     * use Google\Cloud\Bigtable\BigtableClient;
+     * $bigtable = new BigtableClient();
+     * $instanceType = Google\Cloud\Bigtable\Instance::INSTANCE_TYPE_DEVELOPMENT;
+     * $operation = $instance->create(
+     *     'instanceId',
+     *     $instanceType,
+     *     ['foo' => 'bar'],
+     *     [$bigtable->clusterMetadata('my-cluster', 'us-east1-b')]
+     * );
+     * ```
+     *
+     * @codingStandardsIgnoreStart
+     * @see https://cloud.google.com/bigtable/docs/reference/admin/rpc/google.bigtable.admin.v2#CreateInstanceRequest CreateInstanceRequest
+     *
+     * @param string $displayName **Defaults to** the value of $instanceId.
+     * @param int $type Possible values are represented by the following constants: 
+     *            `Google\Cloud\Bigtable\Instance::INSTANCE_TYPE_PRODUCTION`, 
+     *            `Google\Cloud\Bigtable\Instance::INSTANCE_TYPE_DEVELOPMENT` and
+     *            `Google\Cloud\Bigtable\Instance::INSTANCE_TYPE_UNSPECIFIED`.
+     *            **Defaults to** using `Google\Cloud\Bigtable\Instance::INSTANCE_TYPE_UNSPECIFIED`.
+     * @param array $labels as key/value pair ['foo' => 'bar']. For more information, see
+     *        [Using labels to organize Google Cloud Platform resources](https://cloudplatform.googleblog.com/2015/10/using-labels-to-organize-Google-Cloud-Platform-resources.html).
+     * @param array $clustersMetadata [
+     *        $bigtable->clusterMetadata('clusterId1', 'locationId1', 'SSD', 3), 
+     *        $bigtable->clusterMetadata('clusterId2', 'locationId2', 'SSD', 3)
+     * ]
+     * @return LongRunningOperation<Instance>
+     * @codingStandardsIgnoreEnd
+     */
+    public function create($displayName, $type, $labels, $clustersMetadata)
+    {
+        $projectName = InstanceAdminClient::projectName($this->projectId);
+        $instanceId = InstanceAdminClient::parseName($this->name)['instance'];
+        $displayName = ($displayName)
+            ? $displayName
+            : InstanceAdminClient::parseName($this->name)['instance'];
+        $type = ($type) ? $type : self::INSTANCE_TYPE_UNSPECIFIED;
+        $labels = ($labels) ? $labels : [];
+        
+        $clustersArray = [];
+        foreach ($clustersMetadata as $value) {
+            $id = $value['clusterId'];
```

Comment:
Users may not choose to use the helper method on the `BigtableClient`. We *have to* validate this here.